javalib/src/main/scala/java/util/Date.scala

```diff
@@ -0,0 +1,43 @@
+package java.util
+
+/** Ported from Scala JS and Apache Harmony
+ * - omits deprecation
+ * - immutable
+ *
+ * TODO: needs TimeZone, Calendar or java.util.time for compatible toString
+ *       and other deprecated methods if needed
+ */
+class Date(protected[this] val milliseconds: Long)
+    extends Object
+    with Serializable
+    with Cloneable
+    with Comparable[Date] {
+
+  //def this() = this(System.currentTimeMillis())
+  def this() = this(System.nanoTime() / 1000000L)
```

Comment:
Using nanoTime for date is likely going to end up being extremely confusing. I think we really need to implement currentTimeMillis first, and then use it instead.